tests/ui/macros/cfg.stderr

```diff
@@ -6,11 +6,11 @@ LL |     cfg!();
    |
    = note: this error originates in the macro `cfg` (in Nightly builds, run with -Z macro-backtrace for more info)
 
-error: expected identifier, found `123`
+error[E0565]: unsupported literal
```

Comment:
"unsupported literal" is a very basic error message, and doesn't give any idea what kind of literal is allowed. There is `attr_unsupported_literal_cfg_string`, which becomes "literal in `cfg` predicate value must be a string". Could something like that be used to indicate only a boolean literal is allowed?

Reply:
Yes, see the last commit [`f9fc1fb` (#131034)](https://github.com/rust-lang/rust/pull/131034/commits/f9fc1fb5c8af2077328dfbe849d7608189818877).